pkg/topology/topology.go

```diff
@@ -13,10 +14,14 @@ import (
 var ErrNotFound = errors.New("no peer found")
 
 type Driver interface {
-	AddPeer(overlay swarm.Address) error
+	Peerer
 	ChunkPeerer
 }
 
+type Peerer interface {
```

Comment:
I think `AddPeerer` is more in line with what the interface does

Reply:
Or `PeerAdder` ;)

Reply:
Hmm, I am not sure about this. Aren't we adding the peer to the toplogy? And `Peerer` for me is more like a word instead of something like `PeerAdder`. Could you explain, pls?

Reply:
Hahaha, @janos , same time :)

Reply:
Well, according to go conventions, an interface of name `Peerer` would infer a single method whose name would be `Peer()` or potentially `Peers()`. `PeerAdder` then, is better description of what we're passing around

Reply:
Ok, `PeerAdder` it is :)